Sources/Get/Request.swift

```diff
@@ -40,79 +40,165 @@ public struct Request<Response>: @unchecked Sendable {
         self.id = id
     }
 
-    /// Changes the respones type keeping the rest of the request parameters.
+    /// Initialiazes the request with the given parameters.
+    public init(
+        path: String,
+        method: HTTPMethod = .get,
+        query: [(String, String?)]? = nil,
+        body: Encodable? = nil,
+        headers: [String: String]? = nil,
+        id: String? = nil
+    ) {
```

Comment:
This makes sense to me. I was a bit confused that `path` had been renamed to `url` in 1.0, but it makes sense given the dual purpose.. But changing the underlying type to `URL` and having this initialiser sounds good in terms of CreateAPI usage 👍 

Just to confirm though, this is targeting version 2.0.0?

Reply:
Yes, I used to serve both purposes, but both poorly. This should be much clearer.

Yes, I know I just released 1.0, but I want to address these issues earlier. Yes, I'll release it in a 2.0 because there are some breaking changes.

Reply:
Just my opinion, since there are already breaking changes I think it would be okay to just remove the old telescoping methods instead of at a later date. This is a foundational change and developers would have to be responsible for making the 2.0 major update.

Reply:
I agree, I don't want to pollute the APIs, and it's still at the early stage.